Sources/Card/Models/ExpirationDate.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+public struct ExpirationDate: Codable {
+
+    var expiryString: String
+
+    /// Initialize a card expiry object
+    /// - Parameters:
+    ///   - month: The card expiration month (1 to 12)
+    ///   - year: The card expiration year in 4-digit format, `YYYY`
+    public init(month: Int, year: Int) {
+        let isSingleDigit = (1...9).contains(month)
+        let monthString = isSingleDigit ? "0\(month)" : "\(month)"
+        expiryString = "\(year)-\(monthString)"
```

Comment:
It [looks like Android](https://github.com/paypal/Android-SDK/blob/8b86a133ff5aa2cc3027da8d8ba8c70bdffd8e61/Demo/src/main/java/com/paypal/android/ui/card/validation/DateFormatter.kt) used `/` between the month and year and formats as `month/year`. Cool if we update to match?

Reply:
I think that was for their demo app. In here we'd need to format it to `YYYY-MM` as expected by our server

Reply:
Ah, you're right, my bad! That makes sense, sorry missed that that was a UI component.